src/persist/src/indexed/encoding.rs

```diff
@@ -423,8 +407,8 @@ mod tests {
 
     use super::*;
 
-    fn update_with_ts(seqno: u64, ts: u64) -> (SeqNo, (String, String), u64, isize) {
-        (SeqNo(seqno), ("".into(), "".into()), ts, 1)
+    fn update_with_ts(_seqno: u64, ts: u64) -> ((String, String), u64, isize) {
```

Comment:
remove this param from the method

Reply:
ah whoops i thought i removed that before pushing!